packages/cms/src/lokalize/clear-mutations-log.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Clear the key mutation history, and compare the keys in the development and
+ * production datasets to see what items should be deleted from production.
+ */
+
+import { clearLogFile } from './logic';
+
+(async function run() {
+  clearLogFile();
+})().catch((err) => {
+  console.error(`Clear failed: ${err.message}`);
+  process.exit(1);
+});
```

Comment:
nit: called function isn't a promise so we could in theory wrap it in a try/catch to clean up a lot of noisy characters :-)

```suggestion
try {
  clearLogFile();
} catch (err) {
  console.error(`Clear failed: ${err.message}`);
  process.exit(1);
}
```

Reply:
I gave it this format to be consistent with the other cli scripts, and it gives the ability to easily extend it without refactoring.

Reply:
I think I'd prefer to keep it like this for now.